packages/xdl/src/createCompiler.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk';
+import formatWebpackMessages from 'react-dev-utils/formatWebpackMessages';
+import clearConsole from 'react-dev-utils/clearConsole';
+import * as ProjectUtils from './project/ProjectUtils';
+
+const isInteractive = true; //process.stdout.isTTY;
+const CONSOLE_TAG = 'expo';
+
+function logToConsole(message) {
+  // Prevent double messaging
+  if (!process.env.DEBUG) {
+    console.log(message);
+  }
+}
+
+function log(projectRoot, message) {
+  logToConsole(message);
+  ProjectUtils.logDebug(projectRoot, CONSOLE_TAG, message);
+}
+
+function logWarning(projectRoot, message) {
+  logToConsole(message);
```

Comment:
```suggestion
```
This line is not needed. `ProjectUtils.logWarning` will log the message to console as well, so now we're logging everything twice.

Reply:
The format seems to be broken when I use chalk with `ProjectUtils.logWarning` or `ProjectUtils.logInfo`. I can't reset the style after using underline, and newlines seem to be trimmed.